plugins/org.jkiss.dbeaver.ui.dashboard/src/org/jkiss/dbeaver/ui/dashboard/view/DashboardManagerDialog.java

```diff
@@ -198,13 +198,22 @@ public void update(ViewerCell cell) {
             GridData gd = new GridData(GridData.FILL_VERTICAL);
             buttonBar.setLayoutData(gd);
 
-            newButton = UIUtils.createPushButton(buttonBar, UIDashboardMessages.dialog_dashboard_manager_button_new, null, new SelectionAdapter() {
+            newDatabaseChartButton = UIUtils.createPushButton(buttonBar, UIDashboardMessages.dialog_dashboard_manager_button_database_new, null, new SelectionAdapter() {
```

Comment:
Do not create new labels for each dashboard type. Use label `New ${dash.type.name}` where dash.type.name is taken from dashboard registry.
Iterate over all dashboard providers DashboardProviderDescriptor with supportsCustomDashboards==true